scripts/ci/images/Dockerfile.ci-spack-ubuntu20.04-base

```diff
@@ -11,6 +11,14 @@ RUN . /spack/share/spack/setup-env.sh && \
     spack config add "config:checksum:false" && \
     spack config add "config:build_jobs:$(nproc)" && \
     spack config add "concretizer:unify:false" && \
+    spack env create adios2-ci-mpich && \
```

Comment:
Why has this to be moved up?

Reply:
Because it failed when it was done at the end.  I have no idea why it works when done first, and didn't take the time to investigate.